packages/mdc-tabs/tab-bar-scroller/component.ts

```diff
@@ -21,58 +21,63 @@
  * THE SOFTWARE.
  */
 
-import {getCorrectPropertyName} from '@material/animation/index.ts';
+import {getCorrectPropertyName} from '@material/animation/index';
 import {MDCComponent} from '@material/base/component';
+import {MDCTabBar, MDCTabBarFactory} from '../tab-bar';
```

Comment:
i know in #4407 we mentioned something to add `/index`. Not sure what we decided, but we should be consistent with whatever we chose here.